network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -12,6 +12,15 @@ The Network Observability Operator enables administrators to observe and analyze
 These release notes track the development of the Network Observability Operator in the {product-title}.
 
 For an overview of the Network Observability Operator, see xref:../network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
+[id="network-observability-operator-release-notes-1-4-2"]
+== Network Observability Operator 1.4.2
+The following advisory is available for the Network Observability Operator 1.4.2:
+//NOTE: this link will be broken until the live errata is available on 1.4.2 GA, 10/30/23
+* link:https://access.redhat.com/errata/RHSA-2023:122668[2023:122668 Network Observability Operator 1.4.2]
```

Comment:
Need a blank line before line 19 so it puts the bullet item on the next line

Reply:
Thanks Steven. I noticed this and figured I'll fix it when I get the live ID for errata.